app/views/shared/menu/_cart_sidebar.html.haml

```diff
@@ -22,7 +22,16 @@
           %td.text-right
             %span.quantity {{ line_item.quantity }}
           %td
-            %span.total-price.right {{ line_item.total_price | localizeCurrency }}
+            .total-price.text-right {{ line_item.total_price | localizeCurrency }}
+            .flex.unit-price
+              %div{:style => "margin-right: 5px"}
+                %question-mark-with-tooltip{"question-mark-with-tooltip" => "_",
+                "question-mark-with-tooltip-append-to-body" => "true",
+                "question-mark-with-tooltip-placement" => "top",
+                "question-mark-with-tooltip-animation" => true,
+                context: "'cart-sidebar'"}
+              .options-text
+                {{ line_item.variant.unit_price_price | localizeCurrency }} / {{ line_item.variant.unit_price_unit }}
```

Comment:
I missed that `unit_price_price`. I think it should be changed to `unit_price` as well.

Reply:
As I commented here https://github.com/openfoodfoundation/openfoodnetwork/pull/6905#discussion_r578401368 I wanted to have such thing like : `unit_price.price` and `unit_price.unit`.